### PR TITLE
Suggestions

### PR DESCRIPTION
**Pull request by jcinnamond**
I'm not suggesting you must merge these, but here are some changes I thought about while reading the cheat sheet.

Also, it might be worth clarifying that you _can_ repeat words that other panellists have used on a topic, but I didn't add that as I'm not completely sure that it's the case.

### CODE REVIEW

File: docs/game/cheat_sheet.md [hunk 1]
```diff
@@ -12,37 +12,42 @@ enjoyable, entertaining and informative show. Winning isn't everything.
...
+  * *buzz* "repetition of [word]"
+  * *buzz* "deviation (followed by why you think they have deviated)"
 
 Principal Rules
 ---------------
 
-Here's the three main rules of Just A Minute, and a few guidelines on when you
-can and can't challenge.
+The rules of Just A Minute are simple. Each panellist must attempt to
+speak on a given topic for three minutes without hesitation,
```
Comment (AJFaraday) [inline docs/game/cheat_sheet.md]: Factual error. One minute.

File: docs/game/cheat_sheet.md [hunk 2]
```diff
@@ -57,16 +62,21 @@ Tactics
...
 
+Whatever tactic you employ, doing so with a sense of fun will win you
+the love of the audience, even if it doesn't win you the point.
+
 Research
 --------
 
 Depending how recently it has been broadcast, you can hear actual shows on the BBC at: http://www.bbc.co.uk/programmes/b006s5dp/episodes/player
 
+Some episodes are also available on YouTube, for example https://www.youtube.com/watch?v=D87DN_15r5Q
```
Comment (AJFaraday) [inline docs/game/cheat_sheet.md:79]: Good point

File: docs/game/cheat_sheet.md [hunk 3]
```diff
@@ -12,37 +12,42 @@ enjoyable, entertaining and informative show. Winning isn't everything.
 Manners
 -------
 
-Some general manners guidelines for panel games, and specifically Just A Minute
+Panel games, much like the greatness of Great Britain, are founded on
+good manners. Here are some guidelines that make Just A Minute flow smoothly.
```
Comment (AJFaraday) [inline docs/game/cheat_sheet.md]: We may need to tone-down the patriotism, many of the intended consumers are from other nations.

File: docs/game/cheat_sheet.md [hunk 4]
```diff
@@ -12,37 +12,42 @@ enjoyable, entertaining and informative show. Winning isn't everything.
...
 
 * Not a word until round one. The introduction primes the audience for the show, this is easiest to do with one voice.
 * Good humour is the most important ingredient.
-** Avoid speaking over each other, listening to two voices is hard work.
+* Avoid speaking over each other, listening to two voices is hard work.
 * When the timer is running, one speaker is being featured. Listen closely.
 * There is often some argument following challenges.
   * This is part of the game but KEEP IT FRIENDLY!
-  * The hosts word goes. This isn't just megalomania, it helps to keep the game moving.
+  * The host's word goes. This isn't just megalomania, it helps to keep the game moving.
```
Comment (AJFaraday) [inline docs/game/cheat_sheet.md]: The host is word goes?!?

File: docs/game/cheat_sheet.md [hunk 5]
```diff
@@ -12,37 +12,42 @@ enjoyable, entertaining and informative show. Winning isn't everything.
...
 * Any lengthy pause
 * Any hedging sound (erm, um, ahhh etc.)
 * Slowing down until they're almost stopped.
-* Let people take a breath after sentences.
+* But! Let people take a breath after sentences.
 
 Repetition:
 
 * Repeating any word that they have said during that round.
+* Repeating any word that they have said during that round.
```
Comment (AJFaraday) [inline docs/game/cheat_sheet.md]: Puzzled by this diff, did you duplicate this line?

File: docs/game/cheat_sheet.md [hunk 6]
```diff
@@ -57,16 +62,21 @@ Tactics
...
+* It ~can~ be legitimate to leave the topic if you re-define it early on (e.g. "I've got a dog called Single Responsibility Principal..."), but if you go too far this is likely to be challenged.
+* If you don't know about a topic, you could try talking about the fact that you know nothing about it, or about knowing it very well (instead of actually talking about it). This probably won't last the minute, but it's fairly entertaining and better than simply stopping.
 * Using lists is a common strategy.
   * This can be risky, you risk repeating yourself, or repeating the word 'and' too many times.
   * Lists of synonyms can be used in most sentences. E.g. "I learned everything I know about this topic from my mentor, adviser, teacher, confidant, role model... "
   * Most topics could lead to a list. E.g. with the topic of "How I like to work" you could say "I like to work remotely whilst travelling between various cities such as London, Paris, Milan..."
 
+Whatever tactic you employ, doing so with a sense of fun will win you
+the love of the audience, even if it doesn't win you the point.
```
Comment (AJFaraday) [inline docs/game/cheat_sheet.md:72]: Important point, well made.

File: docs/game/cheat_sheet.md (hunk 4 above)
Comment (jcinnamond) [inline docs/game/cheat_sheet.md]: Possessive. The word belonging to the word. (As in, not many hosts and then a word.)

Comment (jcinnamond) [inline docs/game/cheat_sheet.md]: ffs. The word belongs to the host.

File: docs/game/cheat_sheet.md (hunk 5 above)
Comment (jcinnamond) [inline docs/game/cheat_sheet.md]: I... (wait for it)... _repeated_ it. Such funny.

File: docs/game/cheat_sheet.md (hunk 3 above)
Comment (jcinnamond) [inline docs/game/cheat_sheet.md]: Indeed. I was aiming for a play on the quintessential Britishness of the game, and perhaps also on it's host. But this falls a little flat.

Comment (jcinnamond) [inline docs/game/cheat_sheet.md]: How about:

```
Panel games, like so many things in life, are founded on good manners.
```

Or perhaps I should just remove it. 

I initially found it a bit heavy handed that a cheat sheet starts with a guide to good manners, but it's actually a really useful and important section. Lightening the tone is hard.

Comment (AJFaraday) [inline docs/game/cheat_sheet.md]: That's a definite improvement. 

My thinking here is that this is also a cheat-sheet for presenting as part of a panel, not just playing the game. If we can't speak in turn and present a clear narrative to the audience then it doesn't matter who wins, it's a really messy segment.

File: docs/game/cheat_sheet.md (hunk 4 above)
Comment (AJFaraday) [inline docs/game/cheat_sheet.md]: Case in point, Parsons Green https://en.wikipedia.org/wiki/Parsons_Green.

To my view, if we say Parson's Green, it seems very much like we are suggesting that the beloved host is, in fact, green, but Parsons Green makes it clear that the Green belongs to the parson.

**Pull request merged**